File: BY-PYTHON/chekalphabet.py
def is_alphabet(ch):
    if (ch >= 'a' and ch <= 'z') or (ch >= 'A' and ch <= 'Z'):
        print("alphabet")
        
    else: 
        print("not alphabet")
        
        
        
ch = input("Enter any character: ")
x = is_alphabet(ch)
print(x)

File: JAVA/chekalphabet.java
import java.util.Scanner;


public class chekalphabet {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Enter  a character : ");
        char  ch = sc.next().charAt(0);



        if (Character.isLetter(ch)) {
            System.out.println(ch + " is an alphabet.");
        }

        else {
            System.out.println(ch + " is not an alphabet.");
            
            
        }
    }
}